app/upgrades/v4/prop12.go

```diff
@@ -10,7 +10,7 @@ import (
 	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
 )
 
-func Prop12(ctx sdk.Context, bank *bankkeeper.Keeper, distr *distrkeeper.Keeper) {
+func Prop12(ctx sdk.Context, bank bankkeeper.Keeper, distr *distrkeeper.Keeper) {
```

Comment:
Why not distrkeeper too?

Reply:
Because we alter the distr modules fields in lines 35-37

Reply:
hmm... okay? but we do that through method calls, so it should be the same as bank right? Like with both modules we're using method calls that cause mutations, but we're not doing any direct modifications by setting fields directly?

It's been a while since I've thought seriously about golang call by reference / call by value differences, so I'm probably just not getting it. (I'm spoiled by Haskell / FP where everything is immutable lol)